packages/react-aria-components/docs/Calendar.mdx

```diff
@@ -699,6 +711,186 @@ function MyCalendarCell(props) {
 }
 ```
 
+### Contexts
+
+All React Aria Components export a corresponding context that can be used to send props to them from a parent element. This enables you to build your own compositional APIs similar to those found in React Aria Components itself. You can send any prop via context that you could pass to the corresponding component, and refs are supported as well. The local props and ref on the component are merged with the ones passed via context, with the local props taking precedence (following the rules documented in [mergeProps](mergeProps.html)).
+
+<ContextTable components={['Calendar']} docs={docs} />
+
+This example uses `CalendarContext` to create a composite component containing a calendar and buttons representing preset dates.
+
+```tsx example render=false export=true
+import {CalendarContext, useSlottedContext} from 'react-aria-components';
+
+function CalendarPicker({children}) {
+  let [value, onChange] = React.useState(null);
+  let [focusedValue, onFocusChange] = React.useState(null);
+
+  return (
+    /*- begin highlight -*/
+    <CalendarContext.Provider value={{value, onChange, focusedValue, onFocusChange}}>
+    {/*- end highlight -*/}
+      <div className="calendar-picker">
+        {children}
+      </div>
+    </CalendarContext.Provider>
+  );
+}
+
+interface PresetProps {
+  date: CalendarDate,
+  children: React.ReactNode
+}
+
+function Preset({date, children}: PresetProps) {
+  /*- begin highlight -*/
+  let context = useSlottedContext(CalendarContext);
```

Comment:
Can we add a TypeLink + description for `useSlottedContext` as well? Right now it isn't clear in the docs to the end user what the difference is between this and `useContextProps`

Reply:
Yeah I wanted to avoid having to explain all about slots in this one example which doesn't even use them, but I guess.